<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGBA::ForwarderView</class>
 <widget class="QDialog" name="QGBA::ForwarderView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create forwarder</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Files</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>ROM file:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="romFilename"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="romBrowse">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Output filename:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="outputFilename"/>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="outputBrowse">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Forwarder base:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="baseType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Latest stable version</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Latest development build</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Specific file</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="baseLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Base file:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="baseFilename">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QPushButton" name="baseBrowse">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>System</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item alignment="Qt::AlignHCenter">
       <widget class="QRadioButton" name="system3DS">
        <property name="text">
         <string>3DS</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">system</string>
        </attribute>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QRadioButton" name="systemVita">
        <property name="text">
         <string>Vita</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">system</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Presentation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Title:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="title"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="imagesLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Images:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="imageSelect">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="useDefaultImage">
          <property name="text">
           <string>Use default image</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,1,0" columnstretch="0,1,0">
        <item row="0" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="imagePreview">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>128</width>
            <height>128</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1,0,1,0">
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="preferredLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Preferred size:</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="preferredWidth">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string notr="true">0</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="preferredX">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string notr="true">×</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="preferredHeight">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string notr="true">0</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="imageBrowse">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Select image file</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QProgressBar" name="progressBar">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QGBA::ForwarderView</receiver>
   <slot>deleteLater()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="system"/>
 </buttongroups>
</ui>
